src/main/java/pl/knpj/servlet/dao/QuestionDAO.java

```diff
@@ -0,0 +1,91 @@
+package pl.knpj.servlet.dao;
+
+import pl.knpj.servlet.model.Answer;
+import pl.knpj.servlet.model.Question;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Iterator;
+
+/**
+ * Created by vadikms on 29.11.16.
+ */
+public class QuestionDAO extends BaseDAO{
+
+    private static final String GET_QUESTION_SQL =
+            "SELECT id, title, description " +
+                    "FROM \"questions\" WHERE title = ?";
+    private static final String GET_ANSWERS_SQL =
```

Comment:
Ale forKey na nazwe kolumny jest raczej srednie
Wszystko powinienes nazywac tak aby od razu bylo wiadomo co to jest